.github/workflows/destroy.yml

```diff
@@ -0,0 +1,31 @@
+name: Destroying Terraform Resources GH Actions
+run-name: Destroying Infrastructure
+
+on:
+  workflow_dispatch:
+
+jobs:
+  destroy_infra:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v4
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws-session-token: ${{ secrets.AWS_SESSION_TOKEN }}
+          aws-region: us-east-1
+
+      - name: Setup Terraform
+        uses: hashicorp/setup-terraform@v3
+
+      - name: Terraform Init
+        run: terraform init
+        working-directory: Terraform
```

Comment:
Why use working-directory here ? its being used multiple time. Lets use env

Reply:
working-directory is used to avoid writing command to change into the Terraform directory like cd Terraform & terraform init. Using working-directory, the runner will go into that directory and run the specified command. 
I have now updated the code to use environment variable so that I don't have to specify the same working-directory in multiple steps.